doc/tools/make_rst.py

```diff
@@ -1438,5 +1439,24 @@ def sanitize_operator_name(dirty_name, state):  # type: (str, State) -> str
     return clear_name
 
 
+def indent_bullets(text):  # type: (str) -> str
+    # Take the text and check each line for a bullet point represented by "-".
+    # Where found, indent the given line by a further "\t".
+    # Used to properly indent bullet points contained in the description for enum values.
+    # Ignore the first line - text will be prepended to it so bullet points wouldn't work anyway.
+    bullet_points = "-"
+
+    lines = text.splitlines(keepends=True)
+    for line_index, line in enumerate(lines[1:]):
+        pos = 0
+        while pos < len(line) and line[pos] == "\t":
+            pos += 1
+
+        if pos < len(line) and line[pos] in bullet_points:
+            lines[line_index + 1] = line[:pos] + "\t" + line[pos:]
```

Comment:
If done like that then `line_index` is not really the actual line index (you have to remember to add one) so it's misleading (yeah, I'm nitpicking). I'd go with:
```suggestion
    for line_index, line in enumerate(lines[1:], start=1):
        pos = 0
        while pos < len(line) and line[pos] == "\t":
            pos += 1

        if pos < len(line) and line[pos] in bullet_points:
            lines[line_index] = line[:pos] + "\t" + line[pos:]
```
or:
```suggestion
    for line_index, line in enumerate(lines):    
        if line_index == 0:
            continue
        while pos < len(line) and line[pos] == "\t":
            pos += 1

        if pos < len(line) and line[pos] in bullet_points:
            lines[line_index] = line[:pos] + "\t" + line[pos:]
```